clients/python/lakefs_client/client.py

```diff
@@ -27,18 +29,15 @@ def __init__(self, configuration=None, header_name=None, header_value=None, cook
             configuration = LakeFSClient._ensure_endpoint(configuration)
         self._api = _WrappedApiClient(configuration=configuration, header_name=header_name,
                                           header_value=header_value, cookie=cookie, pool_threads=pool_threads)
-        self.actions = ActionsApi(self._api)
-        self.auth = AuthApi(self._api)
-        self.branches = BranchesApi(self._api)
-        self.commits = CommitsApi(self._api)
-        self.config = ConfigApi(self._api)
-        self.health = HealthCheckApi(self._api)
-        self.metadata = MetadataApi(self._api)
-        self.objects = ObjectsApi(self._api)
-        self.refs = RefsApi(self._api)
-        self.repositories = RepositoriesApi(self._api)
-        self.staging = StagingApi(self._api)
-        self.tags = TagsApi(self._api)
+        for key, value in inspect.getmembers(sys.modules['lakefs_client.apis'], inspect.isclass):
+            name = key.lower()
+            if not name.endswith('api'):
+                continue
+            api_instance = value(self._api)
+            attr_name = name[0:len(name)-3]
+            if attr_name not in keyword.kwlist:
+                setattr(self, attr_name, api_instance)
```

Comment:
Why not
```suggestion
            setattr(self, attr_name, api_instance)
```
?

If it _is_ a keyword then you cannot write down the access... but it will still be an attribute and you can still use it with getattr or whatever.  I am not sure why we should prevent that.

Reply:
removed the keyword check - add them both.